Assets/MixedRealityToolkit/Inspectors/ServiceInspectors/HandJointServiceInspector.cs

```diff
@@ -0,0 +1,271 @@
+            ShowHandJointFoldout = EditorGUILayout.Foldout(ShowHandJointFoldout, "Visible Hand Joints");
+            SessionState.SetBool(ShowHandJointFoldoutKey, ShowHandJointFoldout);
+
+            if (ShowHandJointFoldout)
+            {
+                #region setting buttons
+
+                EditorGUILayout.BeginHorizontal();
+
+                if (GUILayout.Button("All"))
+                {
+                    foreach (TrackedHandJoint joint in Enum.GetValues(typeof(TrackedHandJoint)))
+                    {
+                        if (joint == TrackedHandJoint.None)
+                        {
+                            continue;
+                        }
+
+                        SessionState.SetBool(showHandJointSettingKeys[joint], true);
+                        showHandJointSettings[joint] = true;
+                    }
+                }
+
+                if (GUILayout.Button("Fingers"))
+                {
+                    foreach (TrackedHandJoint joint in Enum.GetValues(typeof(TrackedHandJoint)))
+                    {
+                        bool setting = false;
+                        switch (joint)
+                        {
+                            case TrackedHandJoint.IndexTip:
+                            case TrackedHandJoint.IndexDistalJoint:
+                            case TrackedHandJoint.IndexKnuckle:
+                            case TrackedHandJoint.IndexMetacarpal:
+                            case TrackedHandJoint.IndexMiddleJoint:
+
+                            case TrackedHandJoint.MiddleTip:
+                            case TrackedHandJoint.MiddleDistalJoint:
+                            case TrackedHandJoint.MiddleKnuckle:
+                            case TrackedHandJoint.MiddleMetacarpal:
+                            case TrackedHandJoint.MiddleMiddleJoint:
+
+                            case TrackedHandJoint.PinkyTip:
+                            case TrackedHandJoint.PinkyDistalJoint:
+                            case TrackedHandJoint.PinkyKnuckle:
+                            case TrackedHandJoint.PinkyMetacarpal:
+                            case TrackedHandJoint.PinkyMiddleJoint:
+
+                            case TrackedHandJoint.RingTip:
+                            case TrackedHandJoint.RingDistalJoint:
+                            case TrackedHandJoint.RingKnuckle:
+                            case TrackedHandJoint.RingMetacarpal:
+                            case TrackedHandJoint.RingMiddleJoint:
+
+                            case TrackedHandJoint.ThumbTip:
+                            case TrackedHandJoint.ThumbDistalJoint:
+                            case TrackedHandJoint.ThumbMetacarpalJoint:
+                            case TrackedHandJoint.ThumbProximalJoint:
+                                setting = true;
+                                break;
+
+                            default:
+                                break;
```

Comment:
Thoughts on adding an assert/log if we get any other hand joint? This would be a long way in the future but I guess if the platform ever added any more high fidelity joint locations, it might be interesting to get an explicit assert/warning to know that this section needs to be fixed up.

It's also possible that I don't have a fundamentally sound understanding of human anatomy and that this is the exhaustive list of joints, so let me know which it is.